### ISSUE TITLE
Update rollbar.js to latest (from 2.26.1 to 2.26.4)

### ISSUE THREAD

**Issue by jeppeliisberg** [rollbar/rollbar-gem#1158]
rollbar.js is added to my JS bundle by this gem, but it's not updated to the latest version.

**Comment by dkniffin** [comment]
I'm also seeing this warning in Rollbar

![image](https://github.com/user-attachments/assets/e1b91e7c-722e-48f0-a96f-add72e7929ca)

@waltjones can we get this resolved soon?

**Comment by shadoath** [comment]
Looks like Rollbar is now [v3.1.0](https://github.com/rollbar/rollbar.js/releases/tag/v3.1.0) is this gem planning to keep up with that?

**Comment by mateuszgorniak** [comment]
Yep, it becomes really outdated - isn't there a way to connect release updates with updating rollbar.js somehow? It really concerns me, as a user of this product paying for a premium subscription, that the gem is not very well maintained.

**Comment by shadoath** [comment]
Looks like this gem has been updated! 🥳 
https://github.com/rollbar/rollbar-gem/pull/1201

**Comment by zdavis-rollbar** [comment]
@shadoath it has.  Sorry missed this issue.
